kafka-minion/values.yaml

```diff
@@ -119,6 +121,14 @@ serviceMonitor:
   # different matchLabels configuration, to make sure it matches this ServiceMonitor.
   # If this is defined, the release label of the service monitor will match kube-eagle's one
   # additionalLabels:
+  # RelabelConfigs to add to the ServiceMonitor.
+  relabelings:
+  - sourceLabels: [__meta_kubernetes_pod_label_kafka_cr]
```

Comment:
I wouldn't add this to the default values yaml, but rather provide these settings from Supertubes.